modules/troubleshooting-openshift-install-command-issues.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting-installations.adoc
+
+[id="troubleshooting-openshift-install-command-issues_{context}"]
+= Troubleshooting 'openshift-install' command issues
```

Comment:
Same about backticks

Reply:
Resolved for headings throughout the PR.